mlir/utils/performance/createPerformanceReports.py

```diff
@@ -8,53 +8,47 @@
 import sys
 
 #Create html reports from .csv files
-def printAllPerformance(chip):
-    def readReport(file, kind):
-        try:
-            return pd.read_csv(chip + '_' + file)
-        except FileNotFoundError:
-            print(f'${kind} report not found.')
-            return None
-
-    COLUMNS_TO_AVERAGE = ['MLIR TFlops', 'MIOpen TFlops (no MLIR Kernels)', 'MLIR/MIOpen']
-
-    df = readReport(reportUtils.PERF_REPORT_FILE, 'Perf')
-    if df.empty:
+def printAllPerformance(chip, lib='rocBLAS'):
+    perfReportFound = False
+
+    try:
+        df = pd.read_csv(chip + '_' + reportUtils.PERF_REPORT_FILE[lib])
+        perfReportFound = True
+        if 'Tuned MLIR TFlops' in df:
+            COLUMNS_TO_AVERAGE = ['MLIR TFlops', 'Tuned MLIR TFlops',
+                f'{lib} TFlops (no MLIR Kernels)', 'Tuned/Untuned',
+                f'MLIR/{lib}', f'Tuned/{lib}']
+        else:
+            COLUMNS_TO_AVERAGE = ['MLIR TFlops',
+                f'{lib} TFlops (no MLIR Kernels)',
+                f'MLIR/{lib}']
+    except FileNotFoundError:
+        print('Perf report not found.')
         return
-    tuned_df = readReport(reportUtils.MIOPEN_TUNED_REPORT_FILE,
-                          'MIOpen with tuned MLIR')
-    untuned_df = readReport(reportUtils.MIOPEN_UNTUNED_REPORT_FILE,
-                            'MIOpen with untuned MLIR')
-
-    # Add tuned and untuned performance to the existing performance table
-    if not tuned_df.empty and not untuned_df.empty:
-        df['MIOpen TFlops (Tuned MLIR Kernels)'] = tuned_df['TFlops']
-        df['MIOpen TFlops (Untuned MLIR Kernels)'] = untuned_df['TFlops']
-        df['Tuned/Untuned'] = df['MIOpen TFlops (Tuned MLIR Kernels)']/df['MIOpen TFlops (Untuned MLIR Kernels)']
-        df['Tuned/MIOpen'] = df['MIOpen TFlops (Tuned MLIR Kernels)']/df['MIOpen TFlops (no MLIR Kernels)']
-        df.to_csv(chip + '_' + reportUtils.PERF_REPORT_FILE, index=False)
-        COLUMNS_TO_AVERAGE = ['MLIR TFlops', 'MIOpen TFlops (no MLIR Kernels)', 'MLIR/MIOpen',
-                              'MIOpen TFlops (Tuned MLIR Kernels)', 'MIOpen TFlops (Untuned MLIR Kernels)',
-                              'Tuned/Untuned', 'Tuned/MIOpen']
-
-    plotMean = df[COLUMNS_TO_AVERAGE].agg(reportUtils.geoMean)
+
+    # Only plot the actual averages, not the ratios
+    # (This conveniently keeps the old behavior for the no tuning DB case)
+    plotMean = df[COLUMNS_TO_AVERAGE[:3]].agg(reportUtils.geoMean)
     plotMean.name = "Geo. mean"
     plotMean = pd.DataFrame(plotMean).T
 
-    plotMean[['MLIR TFlops', 'MIOpen TFlops (no MLIR Kernels)']]\
-        .to_csv(chip + '_' + reportUtils.PERF_PLOT_REPORT_FILE, index=False)
+    plotMean[['MLIR TFlops', f'{lib} TFlops (no MLIR Kernels)']]\
+        .to_csv(chip + '_' + reportUtils.PERF_PLOT_REPORT_FILE[lib], index=False)
 
-    means = df.groupby(["Direction", "DataType", "InputLayout"])[COLUMNS_TO_AVERAGE]\
+    means = df.groupby(["DataType"])[COLUMNS_TO_AVERAGE]\
```

Comment:
More specifically, even though this is now a unified file, could we please include "Direction" and "InputLayout" when they exist? That's an important breakdown.